src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx

```diff
@@ -13,81 +13,73 @@ import CoursesFiltersDrawer from '~/containers/my-courses/courses-filters-drawer
 
 import { authRoutes } from '~/router/constants/authRoutes'
 
-import useForm from '~/hooks/use-form'
 import { useDrawer } from '~/hooks/use-drawer'
 import useBreakpoints from '~/hooks/use-breakpoints'
-import { useDebounce } from '~/hooks/use-debounce'
 
-import { CourseFilters } from '~/types'
+import { CourseFilters, CourseFiltersActions } from '~/types'
 import { styles } from '~/containers/my-courses/add-course-with-input/AddCourseWithInput.styles'
 
 interface AddCoursesWithInputProps {
-  fetchData: () => Promise<void>
-  searchRef: MutableRefObject<string>
+  additionalParams: Record<string, unknown>
```

Comment:
Maybe instead of `unknown` type use those which can be there?
As i saw in props there can be those values?
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/105641724/a89ef422-c256-4f73-8118-e2c4fab0c4b6)

Reply:
Fixed.
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104922087/8efc4610-9c4e-402d-bb42-6d2e9d1ff758)